src/nbl/asset/utils/CIESProfile.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_ASSET_C_IES_PROFILE_H_INCLUDED__
+#define __NBL_ASSET_C_IES_PROFILE_H_INCLUDED__
+
+#include "nbl/core/Types.h"
+#include <sstream>
+
+namespace nbl 
+{
+    namespace asset 
+    {
+        class CIESProfile 
+        {
+            public:
+                using IES_STORAGE_FORMAT = double;
+
+                _NBL_STATIC_INLINE_CONSTEXPR size_t CDC_DEFAULT_TEXTURE_WIDTH = 1024;
+                _NBL_STATIC_INLINE_CONSTEXPR size_t CDC_DEFAULT_TEXTURE_HEIGHT = 1024;
+
+                _NBL_STATIC_INLINE_CONSTEXPR IES_STORAGE_FORMAT MAX_VANGLE = 180.0;
+                _NBL_STATIC_INLINE_CONSTEXPR IES_STORAGE_FORMAT MAX_HANGLE = 360.0;
+
+                enum PhotometricType : uint32_t 
+                {
+                    TYPE_NONE,
+                    TYPE_C,
+                    TYPE_B,
+                    TYPE_A,
+                };
+
```

Comment:
extract and store (with enum) the spec version the file conforms to 1995 or 2002

Reply:
resolved in following commit